### PR TITLE
chore: fix typo

### PR DESCRIPTION
**Pull request by carlinmack**
:heart: Thank you for your contribution!

### Description

> Please describe briefly your pull request.

f15917601e94c9c8c52576cfa2044eb4d819aa9b introduced lowercase records in the sidebar 

Image

This PR adds `menu_label` with the correct case. After:

image

The order has been swapped, but I think it is preferable in this order

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [ ] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [ ] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [ ] I've added relevant test cases.
- [ ] I've added relevant documentation.
- [ ] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/howtos/i18n/).
- [ ] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [ ] I've NOT included third-party code (copy/pasted source code or new dependencies).
    * If you have added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Frontend**

- [ ] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines.
- [ ] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines.
- [ ] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines.

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

**Review by utnapischtim — APPROVED** [review]
minor: it would be nice to have a more informative commit message. something like that the menu_label falls back to name

File: invenio_app_rdm/administration/records/records.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ class RecordAdminListView(AdminResourceListView):
     resource_config = "records_resource"
     search_request_headers = {"Accept": "application/vnd.inveniordm.v1+json"}
     title = "Records"
+    menu_label = "Records"
```
Comment (Samk13) [inline invenio_app_rdm/administration/records/records.py:29]: Minor: Could you please mark the introduced `menu_label` for translation?
```suggestion
    menu_label = _("Records")
```

File: invenio_app_rdm/administration/records/records.py [hunk 2]
```diff
@@ -100,10 +101,11 @@ class DraftAdminListView(AdminResourceListView):
     """Configuration for OAI-PMH sets list view."""
 
     api_endpoint = "/user/records"
-    name = "Drafts"
+    name = "drafts"
     resource_config = "records_resource"
     search_request_headers = {"Accept": "application/vnd.inveniordm.v1+json"}
     title = "Drafts"
+    menu_label = "Drafts"
```
Comment (Samk13) [inline invenio_app_rdm/administration/records/records.py:108]:
```suggestion
    menu_label = _("Drafts")
```

**Comment by Samk13** [comment]
LGTM!
Thanks for the quick fix!

File: invenio_app_rdm/administration/records/records.py (hunk 2 above)
Comment (utnapischtim) [reply invenio_app_rdm/administration/records/records.py:108]: i thought that too, but i would keep the translation fix combined in the #2886

Comment (Samk13) [reply invenio_app_rdm/administration/records/records.py:108]: If we can make it in one go, why split it into two changes for something so small? :)

**Review by zzacharo — APPROVED** [review]

**Pull request merged**